config/core/resources/subscription.yaml

```diff
@@ -32,117 +61,11 @@ spec:
-                apiVersion:
-                  type: string
-                  minLength: 1
-                kind:
-                  type: string
-                name:
-                  type: string
-                  minLength: 1
-            subscriber:
-              type: object
-              description: "the subscriber that (optionally) processes events."
-              properties:
-                uri:
-                  type: string
-                  description: "the target URI or, if ref is provided, a relative URI reference that will be combined with ref to produce a target URI."
-                  minLength: 1
-                ref:
-                  type: object
-                  description: "a reference to a Kubernetes object from which to retrieve the target URI."
-                  required:
-                    - apiVersion
-                    - kind
-                    - name
-                  properties:
-                    apiVersion:
-                      type: string
-                      minLength: 1
-                    kind:
-                      type: string
-                      minLength: 1
-                    name:
-                      type: string
-                      minLength: 1
-                    namespace:
-                      type: string
-                      minLength: 1
-            reply:
-              type: object
-              description: "the destination that (optionally) receive events."
-              properties:
-                uri:
-                  type: string
-                  description: "the target URI or, if ref is provided, a relative URI reference that will be combined with ref to produce a target URI."
-                  minLength: 1
-                ref:
-                  type: object
-                  description: "a reference to a Kubernetes object from which to retrieve the target URI."
-                  required:
-                    - apiVersion
-                    - kind
-                    - name
-                  properties:
-                    apiVersion:
-                      type: string
-                      minLength: 1
-                    kind:
-                      type: string
-                      minLength: 1
-                    name:
-                      type: string
-                      minLength: 1
-                    namespace:
-                      type: string
-                      minLength: 1
-            delivery:
-              description: "Subscription delivery options. More information: https://knative.dev/docs/eventing/event-delivery."
-              type: object
-              x-kubernetes-preserve-unknown-fields: true
-        status:
-          type: object
-          x-kubernetes-preserve-unknown-fields: true
-  versions:
-  - name: v1alpha1
-    served: false
-    storage: false
-  - name: v1beta1
-    served: true
-    storage: true
+#      conversionReviewVersions: ["v1beta1", "v1alpha1"]
```

Comment:
@vaikas you did disable the conversion, in this PR: https://github.com/knative/eventing/pull/2595

I guess, we can now enable it ? :thinking: